index.test.js

```diff
@@ -190,6 +201,40 @@ describe('express joi', function () {
     });
   });
 
+  describe('#response', function () {
+    it('should return a 500 when the key is not right for regular', function () {
+      const middleware = mod.response(schema);
+      return getRequester(middleware)
+        .get('/response/one')
+        .expect(500)
+    })
+
+    it('should return a 200 when the key is correct for regular', function () {
+      const middleware = mod.response(schema);
+      return getRequester(middleware)
+        .get('/response/1')
+        .expect(200)
+    })
+
+    it('should pass an error to subsequent handler if it is asked', function () {
+      const middleware = mod.response(schema, {
+        passError: true
+      })
+      return getRequester(middleware)
+        .get('/response/one')
+        .expect(200)
```

Comment:
The 200 here seems incorrect. I think it should be a 500 error if passed using `next` to the default express error handler.  

What is the response text here?

Reply:
ah, no, i updated the app.get call to have a second function. i believe this was because coverage was missing: https://github.com/evanshortiss/express-joi-validation/pull/9/files/1551201f5aece2a588d8f0b6ef3b64273f996edb#diff-275a6743b2ff7d68e3d76f3462691716R70